todo.js

```diff
@@ -33,8 +33,10 @@ const updateBtn = document.querySelector(".update-btn");
 
 // Events listener
 toDobutton.addEventListener("click", addTodo , true);
+toDolist.addEventListener("click" , checkNdelete);
```

Comment:
the delete will be active when we click on the list!! 
it should be active when I click on the delete button